PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/forms/arm/CreateDeploymentForm.java

```diff
@@ -153,6 +156,13 @@ public void run(@NotNull ProgressIndicator indicator) {
                                 ((ElementWrapper<Region>) regionCb.getSelectedItem()).getValue());
                     } else {
                         ResourceGroup rg = ((ElementWrapper<ResourceGroup>) rgNameCb.getSelectedItem()).getValue();
+                        List<ResourceEx<Deployment>> deployments = AzureMvpModel.getInstance()
+                                .getDeploymentByRgName(subs.getSubscriptionId(), rg.name());
+                        boolean isExist = deployments.parallelStream()
+                                .anyMatch(deployment -> deployment.getResource().name().equals(deploymentName));
+                        if (isExist) {
+                            throw new RuntimeException("There has been deployment with the same name.");
```

Comment:
A deployment with the same name already exists.

Reply:
Fixed